addon/components/bs-button.js

```diff
@@ -201,8 +177,7 @@ export default class Button extends Component {
    * @default false
    * @public
    */
-  @defaultValue
-  block = false;
+  @arg block = false;
```

Comment:
If I didn't missed something, `block` property is not needed. The usage in template could be changed to `{{@block}}`. See above.

Reply:
Yes, I did that before, but that broke with a build error that `@block` is a reserved keyword! 😝

Reply:
Maybe that's a good indication that we should rename? E.g. @isBlock` or `@blockLevel`.

Reply:
But it's a public argument, so that would be a breaking change!